docs/disaster-recovery.md

```diff
@@ -0,0 +1,399 @@
+
+* Create a Ceph pool that is mirroring enabled, adding the section spec.mirroring in the CephBlockPool CR, example:
+
+    ```yaml=1
+    apiVersion: ceph.rook.io/v1
+    kind: CephBlockPool
+    metadata:
+      name: replicapool
+      namespace: rook-ceph
+    spec:
+      replicated:
+        size: 1
+      mirroring:
+        enabled: true
+        mode: image
+        # schedule(s) of snapshot
+        snapshotSchedules:
+          - interval: 24h # daily snapshots
+            startTime: 14:00:00-05:00
+    ```
+
+* Creating replicapool pool on the cluster:
+
+    ```bash
+    kubectl apply -f pool.yaml -n rook-ceph
+    ```
+
+    > ```bash
+    > cephblockpool.ceph.rook.io/replicapool created
+    >
+
+* Repeat the steps on the peer cluster.
+
+>:memo: WARNING: Pool name across the cluster peers should be the same for RBD replication to function.
+
+For more information on CephBlockPool CR, please refer to the [documentation](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md#mirroring).
+
+## Bootstrap Peers
+
+In order for the rbd-mirror daemon to discover its peer cluster, the peer must be registered and a user account must be created. The following steps enable bootstrapping peers to discover and authenticate to each other.
+
+For more details, refer to the official rbd mirror documentation on [how to create a bootstrap peer](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#bootstrap-peers).
+
+## Create RBDMirror CRD
+
+Replication is handled by the rbd-mirror daemon. The rbd-mirror daemon is responsible for pulling image updates from the remote, peer cluster, and applying them to image within the local cluster.
+
+Creation of the rbd-mirror daemon(s) is done through the custom resource definitions (CRDs).
+For more information on how to set up Ceph RBDMirror CRD, refer to [documentation](https://rook.io/docs/rook/master/ceph-rbd-mirror-crd.html).
+
+>:memo: **NOTE**: rbd-mirror assumes the two clusters are named differently.
+
+## Deploy the Volume Replication CRD
+
+Volume Replication Operator follows controller pattern and provides extended APIs for storage disaster recovery. The extended APIs are provided via Custom Resource Definition (CRD).
+
+>:bulb: For more information, please refer to the [volume-replication-operator](https://github.com/csi-addons/volume-replication-operator).
+
+* Deploy the `VolumeReplicationClass` CRD
+
+    ```bash
+    kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/main/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+    ```
+
+    >```bash
+    > customresourcedefinition.apiextensions.k8s.io/volumereplicationclasses.replication.storage.openshift.io created
+    >```
+
+* Deploy the VolumeReplication CRD
+
+    ```bash
+   kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/main/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
+    ```
```

Comment:
use release branch instead of the main branch. add a note to use latest available release